dependabot/update_dependencies_in_pipeline.py

```diff
@@ -188,34 +188,36 @@ def wait_for_current_level_build(level):
         chat_message += 'Following modules dependency PRs have failed checks...' + "\n"
         for module in pr_checks_failed_modules:
             print(module['name'])
-            chat_message += module['name'] + "\n"
+            chat_message += "<" + module[MODULE_CREATED_PR] + "|" + module['name'] + ">" + "\n"
 
     pr_merged_failed_modules = list(
         filter(lambda s: s[MODULE_CONCLUSION] == MODULE_CONCLUSION_PR_MERGE_FAILURE, current_level_modules))
     if len(pr_merged_failed_modules) != 0:
         module_release_failure = True
         chat_message += 'Following modules dependency PRs could not be merged...' + "\n"
         for module in pr_merged_failed_modules:
-            chat_message += module['name'] + "\n"
+            print(module['name'])
+            chat_message += "<" + module[MODULE_CREATED_PR] + "|" + module['name'] + ">" + "\n"
 
     build_checks_failed_modules = list(
         filter(lambda s: s[MODULE_CONCLUSION] == MODULE_CONCLUSION_BUILD_FAILURE, current_level_modules))
     if len(build_checks_failed_modules) != 0:
         module_release_failure = True
         chat_message += 'Following modules timestamped build checks failed...' + "\n"
         for module in build_checks_failed_modules:
-            chat_message += module['name'] + "\n"
+            print(module['name'])
+            chat_message += "<" + module[MODULE_CREATED_PR] + "|" + module['name'] + ">" + "\n"
 
     build_version_failed_modules = list(
         filter(lambda s: s[MODULE_CONCLUSION] == MODULE_CONCLUSION_VERSION_CANNOT_BE_IDENTIFIED, current_level_modules))
     if len(build_version_failed_modules) != 0:
         module_release_failure = True
         chat_message += 'Following modules timestamped build version cannot be identified...' + "\n"
         for module in build_version_failed_modules:
-            chat_message += module['name'] + "\n"
+            print(module['name'])
+            chat_message += "<" + module[MODULE_CREATED_PR] + "|" + module['name'] + ">" + "\n"
 
     if module_release_failure:
-        print(chat_message)
```

Comment:
Lets not remove it, if the messgae has been missed, we don't have anyway to debug the failure

Reply:
I've added a print(module['name']) within each if block

Reply:
Yes, it will just print the module name, won't it? Not what's wrong with it? We can remove the print module name and print the entire chat message